utils/fetchEarthquakes.ts

```diff
@@ -13,27 +13,20 @@ export const fetchEarthquakes = async (): Promise<Earthquakes> => {
   return res.data.features;
 };
 
-/** Fetch significant earthquakes () */
-// This is only returning one for today?? Maybe need to format differently
-export const fetchSignificantEarthquakes = async (): Promise<Earthquakes> => {
-  const res = await axios.get(
-    'https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/significant_day.geojson'
-  );
-
-  return res.data.features;
-};
```

Comment:
I ended up being able to remove the bulk of these requests I wrote: 

- fetchAndProcessEarthquakes,
 - fetchLastWeekTopMagnitudeEarthquakes,
 - fetchAllLastWeekEarthquakes,
 - fetchTodaySignificantEvents,

This also brought down the API requests in total by like half - not including the API requests I was making in some sub-components. But as usual most of these were written as a way to get familiar with the USGS API and see the different things I could throw at it and get back as a result - and making the requests in components as I build just helps me get an idea of what I actually want to use and how to display it.